wpeverest/user-registration
# Fix- Missing attachment for single file upload due to form-filler

### All Submissions:






### Changes proposed in this Pull Request:
When the maximum file upload is set to one ( single file upload ), Use of a form filler to automatically fill the form will result in a missing attachment for the file upload field in the edit profile.

### How to test the changes in this Pull Request:

1. In registration form drag and drop file upload field, and set maximum file upload to 1.
2. Use form filler to fill the form and upload a file in the file upload field.
3. Check if the uploaded file is shown in the edit profile of my account.

### Types of changes:




### Other information:




### Changelog entry

> Fix- Missing attachment for single file upload due to form-filler
